src/libs/kata-types/src/capablities.rs

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use bitmask_enum::bitmask;
+
+#[bitmask(u8)]
+enum Bitmask {
```

Comment:
This pr is to align with the go runtime, you can refer to [capabilities.go](https://github.com/kata-containers/kata-containers/blob/main/src/runtime/virtcontainers/types/capabilities.go). it will not show all the differencies between the hypervisors, we only call this method when sometimes we need to perform some customized actions for different hypervisors.